{3592: 0}
{3592: 0.007031917106360197}
{3592: 0.013948298990726471}
{3592: 0.024541018065065145}
{3592: 0.03435232490301132}
{3592: 0.05690457159653306}
{3592: 0.06688392953947186}
{3592: 0.10064504155889153}
{3592: 0.11078834533691406}
{3592: 0.14514297526329756}
{3592: 0.15542276203632355}
{3592: 0.1932288366369903}
{3592: 0.20659897243604064}
{3592: 0.25640749325975776}
{3592: 0.2699816692620516}
{3592: 0.31926012225449085}
{3592: 0.34326461143791676}
{3592: 0.3903648513369262}
{3592: 0.41516279336065054}
{3592: 0.8840831057168543}
{3592: 1.0040125916711986}
{3592: 1.3289868729189038}
{3592: 1.3360462910495698}
{3592: 14036.445115381386}
{3592: 24807.63288871525}
{3592: 24807.871539980173}
{3592: 24807.943335659336}
{3592: 24808.17021795828}
{3592: 24808.173957529012}
{3592: 29365.11397570884}
{3592: 29365.75928129768}
{3592: 29366.32555712154}
{3592: 33869.63604045473}
{3592: 38312.51544759888}
{3592: 38312.522077613045}
{3592: 38312.528718125075}
Traceback (most recent call last):
  File "ra.py", line 41, in <module>
    vm.verify_full_state()
  File "/ssd//tools/git/RA/vm.py", line 285, in verify_full_state
    self.run()
  File "/ssd//tools/git/RA/vm.py", line 833, in run
    func, num_inputs, num_output, *funcarg = self.mnemonic_to_func(mnemonic)
  File "/ssd//tools/git/RA/vm.py", line 1191, in mnemonic_to_func
    return d[mnemonic]
KeyError: 'CODEDATACOPY'
